reference/orders.v2.oas2.yml

```diff
@@ -5189,6 +5234,56 @@ components:
           shipping_quotes:
             url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2/orders/163/consignments/shipping/64/shipping_quotes'
             resource: /orders/163/consignments/shipping/64/shipping_quotes
+    shippingConsignment_Put:
```

Comment:
this object schema seems not to be used anywhere ?

Reply:
Added this to `orderConsignment_Put`